.vscode/settings.json

```diff
@@ -6,5 +6,8 @@
   "python.testing.unittestEnabled": false,
   "python.testing.pytestEnabled": true,
   "python.testing.cwd": "${workspaceFolder}",
-  "python.linting.pylintEnabled": false
+  "python.linting.pylintEnabled": true,
```

Comment:
I am starting to think we impose too many things on the IDE side. Wdyt about removing it?

Reply:
(I mean removing `.vscode/settings.json`, and perhaps suggesting a config in contributing.md)